docs/manuals/run-deploy/deploy/databricks/index.md

```diff
@@ -0,0 +1,124 @@
+
+    If you don't have a Linux-based machine, you can use a cloud provider as Azure to
+    [create one](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?tabs=ubuntu).
+
+## Running your application on Databricks
+
+The first step consists in running your Taipy application on Databricks.
+Let's consider the following application:
+
+```python linenums="1"
+{%
+include-markdown "../../code_sample/basic_gui_rest_app.py"
+comments=false
+%}
+```
+
+Connect to Databricks and run the previous code in a Databricks Notebook. You should obtain a similar output:
+```
+[2023-06-12 17:17:25,611][Taipy][INFO] Running in 'single_client' mode in notebook environment
+INFO:Taipy:Running in 'single_client' mode in notebook environment
+[2023-06-12 17:17:25,912][Taipy][INFO]  * Server starting on http://127.0.0.1:5000
+INFO:Taipy: * Server starting on http://127.0.0.1:5000
+```
+
+
+As you can see on the output, the Taipy application is running on Databricks and listening on `localhost`
+(127.0.0.1:5000), which is not accessible from your browser. The next section shows you how to expose your application.
+
+
+## Exposing your application
+
+Here is the global architecture of the solution.
+![Global architecture](./images/global-architecture.svg)
+
+Since we can't connect to Databricks directly, we'll use an SSH tunnel between a proxy machine and the
+Databricks machine, enabling them to communicate.
+We'll also install Nginx on the Linux-based machine to route the request from your browser to your application.
+
+In the end, the browser sends a request to the Nginx on the Linux-based machine; the Nginx will redirect
+this request to the SSH tunnel, which will redirect to the Taipy application.
+
+Here is the technical architecture of the solution.
+![Technical architecture](./images/technical-architecture.svg)
+
+!!! Warning
+
+    The Linux-based machine should be accessible from your network and Databricks.
+
+### 1- Configure the proxy using Nginx
+
+We'll install and configure Nginx on the Linux-based machine to enable browser-based communication.
+Once configured, Nginx will redirect all incoming requests on the port HTTP (80) to port 8080.
+
+First, install [Nginx](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/). Then configure
+the redirection by putting the following content in `/etc/nginx/sites-enabled/default` :
+```
+server {
+    listen 80;
+    location / {
+        proxy_pass http://localhost:8080;
+
+        proxy_read_timeout          36000s;
+        proxy_http_version          1.1;
+        proxy_set_header            Upgrade $http_upgrade;
+        proxy_set_header            Connection $connection_upgrade;
+        proxy_set_header            Host $host;
+        proxy_request_buffering     off;
+    }
+}
+```
+
+And restart Nginx: `systemctl restart nginx`.
+
+### 2 - Connect Databricks to the proxy machine
+
+Open Databricks on a first tab, and go to compute and select your cluster.
```

Comment:
"on a first tab" => "in a first tab"

"and go to compute" => "press "Compute"" (?)